src/components/Atoms/Editbutton/index.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import styled from "styled-components";
+import pencil from "../../../assets/svg/pencil.svg";
+import COLOR from "../../../variables/color.js";
+const EditButton = ({ onClick }) => {
+  return (
+    <Styledbutton onClick={onClick}>
+      <HoverCircle />
+      <Img src={pencil} />
+    </Styledbutton>
+  );
+};
+export default EditButton;
+
+const HoverCircle = styled.div`
+  position: absolute;
+  transition: 0.2s;
+  background-color: ${COLOR.LIGHT_GRAY};
+  border-radius: 50%;
+  width: 100%;
+  height: 100%;
+  margin: auto 0;
+`;
+
+const Styledbutton = styled.button`
```

Comment:
```suggestion
const StyledButton = styled.button`
```
typoのミスを防ぐために code spell checkerという拡張機能もおすすめです
https://www.midnightunity.net/vscode-code-spell-checker/